src/pdn/src/via.cpp

```diff
@@ -1512,6 +1512,64 @@ bool ViaGenerator::build(bool bottom_is_internal_layer,
   return true;
 }
 
+bool ViaGenerator::updateCutSpacing(int rows, int cols)
+{
+  // dims bounded by 1 -> 4
+  rows = std::max(1, std::min(rows, 4));
+  cols = std::max(1, std::min(cols, 4));
+  const int min_dim = std::min(rows, cols);
+  const int max_dim = std::max(rows, cols);
+  int adj_cuts = 0;
+  if (min_dim == 1) {
```

Comment:
I get what you are doing here but I think a comment would be helpful.

Reply:
Added some comments